ArduPlane/Attitude.cpp

```diff
@@ -478,7 +478,7 @@ void Plane::calc_nav_yaw_coordinated(float speed_scaler)
     int16_t commanded_rudder;
 
     // Received an external msg that guides yaw in the last 3 seconds?
-    if ((control_mode == &mode_guided || control_mode == &mode_avoidADSB) &&
+    if (control_mode->is_guided() &&
```

Comment:
Same comment that this looks like a preexsisting bug.

Reply:
per devcall: confirmed bug. Remove "control_mode == &mode_avoidADSB"